README.rst

```diff
@@ -22,13 +32,79 @@ How can I contribute?
 The project is just starting, but we want the entire development process, from conception to realization, to be carried
 out in the open. More will be added to this section in the very near future!
 
+
+Running Kolibri
+---------------
+
+This is how we typically set up a development environment.
+Adjust according to your operating system or personal preferences.
+
+#. Check out the repository::
+
+    $ git clone git@github.com:learningequality/kolibri.git
+    $ cd kolibri/
+
+#. Install a virtual environment for development (Python 2 or Python 3, you choose!)::
+
+    $ sudo pip install virtualenvwrapper
+    $ mkvirtualenv --python=python3 kolibri
+    $ workon kolibri
+
+#. Install kolibri as an editable, meaning your installation will point to your git checkout::
+
+    $ pip install -e .
+
+#. Run the development server::
+
+    $ kolibri manage runserver
+
+
+Testing
+-------
+
+Kolibri comes with a test suite based on ``py.test``. To run tests in your
+current environment::
+
+    $ python setup.py test  # alternatively, "make test" does the same
+
+You can also use ``tox`` to setup a clean and disposable environment::
+
+    $ tox -e py3.4  # Runs tests with Python 3.4
+
+To run tests for all environments, lint and documentation tests,
+use simply ``tox``. This simulates what our CI also does.
+
+To run linting tests (pep8 and static code analysis), use ``tox -e lint`` or
+``make lint``.
+
+
+Current Features
+----------------
+
+This is a WIP. Architecture is new, and many things are mentioned with the predecessor KA Lite in mind.
+
+Software architecture
+~~~~~~~~~~~~~~~~~~~~~
+
+* Using py.test
+* Using tox to build py 2.7, 3.4, 3.5 and pypy
+* Automatic linting test (pep8)
+* Static code analysis check (pyflakes)
+* Circle CI
```

Comment:
I think we had switched to Circle previously because of reliability issues with Travis. This may be hoped to be avoided with a better approach to testing, but I assume @aronasorman would like us to still Circle it up?